forge-game/src/main/java/forge/game/ability/effects/CountersMultiplyEffect.java

```diff
@@ -49,7 +49,7 @@ public void resolve(SpellAbility sa) {
             // gameCard is LKI in that case, the card is not in game anymore
             // or the timestamp did change
             // this should check Self too
-            if (gameCard == null || !tgtCard.equalsWithTimestamp(gameCard)) {
+            if (gameCard == null || !tgtCard.equalsWithGameTimestamp(gameCard)) {
```

Comment:
We need to think about when to check for phasedOut either here or as follow-up

e. g. look at this rubbish:

![grafik](https://user-images.githubusercontent.com/8506892/164718882-60defa3a-d488-4648-bbe9-2d38e441ada7.png)

on resolving counters still get doubled :/

Reply:
in Theory, a phased out Perm shouldn't be able to get counters at all, so i think the check should be added to `addCounterInternal`?

Reply:
I think `canReceiveCounters` might be better then don't even need to run Replacements on it first

Reply:
fixed, also Player that left the game are excluded too